### DISCUSSION TITLE
Radar toggle problem

### DISCUSSION THREAD

**Discussion by 4doschris** [carla-simulator/carla#5409] [Q&A]
Hi all!

I have a problem with the built-in radar of Carla. Whenever I run a script, it plots that the radar is generated, but I can not see it. The situation is the same with the manual_control.py, I can not toggle the radar with pressing "G".

Anybody had the same issue?

Thanks in advance,
Krisztián

![image](https://user-images.githubusercontent.com/102667251/166894182-e37bdb5f-ca7d-4020-b56c-4a2261d6238b.png)

**Comment by ihassan96** [comment]
same here
